lib/configfiles/leap_42.1.xml

```diff
@@ -0,0 +1,1474 @@
+						<content><![CDATA[
+user nginx;
+worker_processes  4;
+
+pid /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    access_log  /var/log/nginx/access.log;
+
+    sendfile        on;
+    tcp_nodelay     on;
+
+    keepalive_timeout  65;
+
+    gzip  on;
+
+    include /etc/nginx/conf.d/*.conf;
+    include /etc/nginx/sites-enabled/*;
+}
+]]>
+						</content>
+					</file>
+					<file name="/etc/nginx/fastcgi_params">
+						<content><![CDATA[
+fastcgi_index index.php;
+
+fastcgi_connect_timeout 65;
+fastcgi_send_timeout    180;
+fastcgi_read_timeout    180;
+
+fastcgi_param  QUERY_STRING       $query_string;
+fastcgi_param  REQUEST_METHOD     $request_method;
+fastcgi_param  CONTENT_TYPE       $content_type;
+fastcgi_param  CONTENT_LENGTH     $content_length;
+
+fastcgi_param  SCRIPT_NAME        $fastcgi_script_name;
+fastcgi_param  REQUEST_URI        $request_uri;
+fastcgi_param  DOCUMENT_URI       $document_uri;
+fastcgi_param  DOCUMENT_ROOT      $document_root;
+fastcgi_param  SERVER_PROTOCOL    $server_protocol;
+fastcgi_param  HTTPS              $https if_not_empty;
+
+fastcgi_param  GATEWAY_INTERFACE  CGI/1.1;
+fastcgi_param  SERVER_SOFTWARE    nginx/$nginx_version;
+
+fastcgi_param  REMOTE_ADDR        $remote_addr;
+fastcgi_param  REMOTE_PORT        $remote_port;
+fastcgi_param  SERVER_ADDR        $server_addr;
+fastcgi_param  SERVER_PORT        $server_port;
+fastcgi_param  SERVER_NAME        $server_name;
+
+# PHP only, required if PHP was built with --enable-force-cgi-redirect
+fastcgi_param  REDIRECT_STATUS    200;
+]]>
+						</content>
+					</file>
+					<include>//service[@type='http']/general/commands</include>
+					<command>
+						<visibility mode="false">{{settings.phpfpm.enabled}}
+						</visibility>
+						<visibility mode="false">{{settings.system.mod_fcgid}}
+						</visibility>
+						<content><![CDATA[service php-fcgi restart]]></content>
+					</command>
+					<command><![CDATA[service nginx restart]]></command>
+				</daemon>
+			</service>
+			<!--DNS -->
+			<service type="dns" title="{{lng.admin.configfiles.dns}}">
+				<!--Bind9 -->
+				<daemon name="bind" title="Bind9 nameserver">
+					<install><![CDATA[zypper install bind]]></install>
+					<command><![CDATA[mkdir {{settings.system.bindconf_directory}}]]></command>
+					<command><![CDATA[echo "include \"{{settings.system.bindconf_directory}}froxlor_bind.conf\";" >> /etc/named.conf]]></command>
+					<command><![CDATA[touch {{settings.system.bindconf_directory}}froxlor_bind.conf]]></command>
+					<command><![CDATA[chown named:named {{settings.system.bindconf_directory}}froxlor_bind.conf]]></command>
+					<command><![CDATA[chmod 0600 {{settings.system.bindconf_directory}}froxlor_bind.conf]]></command>
+					<command><![CDATA[systemctl restart named]]></command>
+				</daemon>
+				<daemon name="powerdns" title="PowerDNS via bind-backend">
```

Comment:
see for example https://github.com/Froxlor/Froxlor/blob/master/lib/configfiles/gentoo.xml#L391 and following lines (that's gentoo, needs adjusting)